plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/BuildMojo.java

```diff
@@ -310,6 +388,28 @@ protected void buildAndTag(ServiceHub hub, ImageConfiguration imageConfig)
                 .build();
     }
 
+    protected io.fabric8.maven.core.service.BuildService.BuildServiceConfig getBuildServiceConfigJib() throws MojoExecutionException {
+        return new io.fabric8.maven.core.service.BuildService.BuildServiceConfig.Builder()
```

Comment:
This is not Jib configuration so why the method is called as jib.

Reply:
I am fetching the BuildServiceConfig in case we want to perform jib build

Reply:
But this is an openshift thing, and OpeShift works ith s2i to build the docker image.